### PR TITLE
Update platforms Tested

### PR DESCRIPTION
**Pull request by byrnHDF**

### CODE REVIEW

**Comment by lrknox** [comment]
I'll push more platform updates to this PR today.

**Review by hyoklee — APPROVED** [review]

File: bin/h4vers [hunk 1]
```diff
@@ -353,7 +353,7 @@ if ($HDF4CONFIGCMAKE) {
   my $sub_rel_ver_str = (
      $newver[3] eq ""
      ? sprintf("\"%s\"", "")
-     : sprintf("\"%s\"", "-".$newver[3].", currently under development")
```
Comment (byrnHDF) [inline bin/h4vers:356]: Looks simple enough.

**Review by lrknox — APPROVED** [review]

File: release_notes/RELEASE.txt [hunk 2]
```diff
@@ -25,7 +25,6 @@ CONTENTS
 - New features and changes
   -- Configuration
   -- C Library
-- Support for new platforms and compilers
 - Bugs fixed since HDF 4.2.16
```
Comment (bmribler) [inline release_notes/RELEASE.txt:28]: should be 4.2.16-2 as the one below

Comment (lrknox) [reply release_notes/RELEASE.txt:28]: Hmmm.  There's nothing in the HISTORY.txt file for a 4.2.16-2 release.  I don't remember why there was a 4.2.16-2 patch release, but I wonder if the previous versions in this RELEASE.txt should all be 4.2.16?
   
There are 2 instances of 4.2.16 and one of 4.2.16-2, so I suppose all 3 in this file should be the same.

Comment (derobins) [reply release_notes/RELEASE.txt:28]: The 4.3.0 release should be off the 4.2.16, IIRC. A patch realease technically isn't a real release.

**Review by derobins — APPROVED** [review]

**Pull request merged**